lib/virt_autotest/utils.pm

```diff
@@ -201,15 +201,25 @@ sub check_guest_health {
     return unless is_x86_64 and ($vm =~ /sle|alp/i);
 
     #check if guest is still alive
-    validate_script_output("virsh domstate $vm", sub { /running/ }, proceed_on_failure => 1) if (script_run("virsh list --all | grep \"$vm \"") == 0);
+    my $vmstate = "nok";
+    my $failures = "";
+    if (script_run("virsh list --all | grep \"$vm \"") == 0) {
+        $vmstate = "ok" if (script_run("virsh list --all | grep \"$vm \" | grep running") == 0);
```

Comment:
How about change this line to 
```
$vmstate = "ok" if (script_run("virsh domstate | grep running") == 0);
```

Reply:
The same meaning, the new one is much better.